README.md

```diff
@@ -1,36 +1,135 @@
-## Inputs
+# GCP SLOs
 
-| Name                               | Description                                                                                                                                                                        | Type        | Default | Required |
-| ---------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- | ------- | :------: |
-| __project__                        | Project ID to create alerts in                                                                                                                                                     | `string`    | n/a     |   yes    |
-| __service_name__                   | Display name of the custom service                                                                                                                                                 | `string`    | n/a     |   yes    |
-| __telemetry_resource_name__        | The full name of the resource that defines this service                                                                                                                            | `string`    | n/a     |    no    |
-| __default_user_labels__            | Labels to be set for __all__ alerts                                                                                                                                                | `map(any)`  | {}      |    no    |
-| __fallback_notification_channels__ | NCs to be set for all alerts that don't provide `notification_channels`. Provide the NCs "id" or "display name" (the latter is dependant on the notification_channel_ids variable) | `list(any)` | []      |    no    |
-| __notification_channel_ids__       | To be able to provide channels display name instead of id/name, provide a  be { display_name: name } or output from tf-module-gcp-notification-channels.                           | `list(any)` | []      |    no    |
-| __default_alert_documentation__    | Documentation to be set for all burn-rate alerts.                                                                                                                                  | `string`    | n/a     |    no    |
-| __slos__                           | The list of alert policies configurations.                                                                                                                                         | `list(any)` | n/a     |   yes    |
+For creating SLOs in GCP Monitoring.
 
-## Burn rate alerting
```

Comment:
I see code about burn rate alerts but nothing described in the docs. Are these alerts still created when you setup the SLO?

Reply:
Yes, it creates one burn-rate alert (fast-burn). I've updated the readme, take a look and let me know if its good enough ^^